config/locales/en.yml

```diff
@@ -168,5 +168,10 @@ en:
         options:
           - "Yes"
           - "I might be evicted soon"
+      feel_safe:
+        title: "Do you feel safe where you live?"
+        options:
+          - "Yes"
+          - "Yes but I’m concerned about the safety of someone else"
```

Comment:
You appear to have lost the No/Not Sure answers from the "Have you been evicted" question

Reply:
Ah. Must have happened during the rebase.